meinberlin/apps/ideas/templates/meinberlin_ideas/idea_create_form.html

```diff
@@ -17,6 +17,11 @@
         </nav>
 
         <h1>{% trans 'Submit a new idea for this project'%}</h1>
+
+        {% if form.errors %}
+            <div class="errorlist">{% trans 'An error occurred while evaluating your data. Please check the data you entered again.' %}</div>
+        {% endif %}
+
```

Comment:
error does not trigger for not adding a title, is that expected behaviour? also for the edit forms

Reply:
Yes, that's a good question, and what I meant by that there are different ways that error messages are displayed. In this case, I don't know where these error messages come from, and they're always in english:
![image](https://user-images.githubusercontent.com/17978375/143561355-5934fad2-29d3-4512-b68c-99dcdb416e9d.png)
I will create an issue for this.

Reply:
@sabinammm they are error messages from the browser as that field is required in the template, the browser knows this and will display an error message when it is not filled, I think it only displays when there is no error handling in the code but maybe it does it even when there is? not sure.